maxun-core/src/browserSide/scraper.js

```
@@ -423,373 +423,648 @@ function scrapableHeuristics(maxCountPerPage = 50, minArea = 20000, scrolls = 3,
+                } else {
+                  nextElements.push(
+                    ...Array.from(frameDoc.querySelectorAll(part))
+                  );
+                }
+              }
+            } catch (e) {
+              console.warn(
+                `Cannot access ${element.tagName.toLowerCase()} content:`,
+                e
+              );
+              continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)